packages/wrangler/src/__tests__/kv.test.ts

```diff
@@ -1150,7 +1155,29 @@ describe("wrangler", () => {
           `kv:bulk put --namespace-id some-namespace-id keys.json`
         );
         expect(requests.count).toEqual(1);
-        expect(std.out).toMatchInlineSnapshot(`""`);
+        expect(std.out).toMatchInlineSnapshot(`"Success!"`);
+        expect(std.warn).toMatchInlineSnapshot(`""`);
+        expect(std.err).toMatchInlineSnapshot(`""`);
+      });
+
+      it("should put the key-values in batches of 5000 parsed from a file", async () => {
+        const keyValues: KeyValue[] = new Array(12000).fill({
+          key: "someKey1",
+          value: "someValue1",
+        });
+        writeFileSync("./keys.json", JSON.stringify(keyValues));
+        const requests = mockPutRequest("some-namespace-id", keyValues);
+        await runWrangler(
+          `kv:bulk put --namespace-id some-namespace-id keys.json`
+        );
+        expect(requests.count).toEqual(3);
+        expect(std.out).toMatchInlineSnapshot(`
+          "Uploaded 0 of 12000.
```

Comment:
I think we should land this, but a potential followup could be a percentage? Something like: `Uploaded 7% (1.5k out of 100.3k)`

Reply:
Added an issue to track this https://github.com/cloudflare/wrangler2/issues/555